CMakeLists.txt

```diff
@@ -74,12 +125,26 @@ include_directories(
 add_library(ale-lib SHARED ${SOURCE_DIR}/ale_interface.cpp ${SOURCES})
 set_target_properties(ale-lib PROPERTIES OUTPUT_NAME ale)
 set_target_properties(ale-lib PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+if(UNIX)
+	install(TARGETS ale-lib 
+		DESTINATION ${LIB_INSTALL_DIR})
+endif()
+
 add_executable(ale-bin ${SOURCE_DIR}/main.cpp ${SOURCE_DIR}/ale_interface.cpp ${SOURCES})
 set_target_properties(ale-bin PROPERTIES OUTPUT_NAME ale)
 set_target_properties(ale-bin PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+if(UNIX)
+	install(TARGETS ale-bin
```

Comment:
I like to install the examples as well . But we are not obliged to. I will drop them out from the install.